lib/models/src/test/java/org/sagebionetworks/repo/model/drs/DrsObjectIdTest.java

```diff
@@ -0,0 +1,88 @@
+package org.sagebionetworks.repo.model.drs;
+
+import org.junit.jupiter.api.Test;
+import org.sagebionetworks.repo.model.entity.IdAndVersion;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+
+public class DrsObjectIdTest {
+
+    private IdAndVersion idAndVersion = IdAndVersion.parse(ENTITY_OBJECT_ID);
+    private static final String ENTITY_OBJECT_ID = "syn54321.1";
+
+    @Test
+    public void testCreateDrsObjectIdWithFileHandleId() {
```

Comment:
method can be renamed to testParseWithValidFileHandleId() and we should do asserEquals on Long as fileHandleId is long